app/ts/components/TokenManager/EtherBalance.tsx

```diff
@@ -0,0 +1,50 @@
+
+import { useSignalEffect } from '@preact/signals'
+import { BigNumber, ethers } from 'ethers'
+import { useAsyncState } from '../../library/preact-utilities.js'
+import { useAccount } from '../../store/account.js'
+import { useNetwork } from '../../store/network.js'
+import { useProviders } from '../../store/provider.js'
+import { AsyncText } from '../AsyncText.js'
+
+export const EtherBalance = () => {
+	const { network } = useNetwork()
+	const { address } = useAccount()
+	const providers = useProviders()
+	const { value: query, waitFor } = useAsyncState<BigNumber>()
```

Comment:
Out of curiosity, why do you name it `query` here?

Reply:
the keyword `value` can sometimes get disorientating? i sometimes couldn't differentiate at a glance if it's a signal or asyncState. maybe worth naming it something else by default?

FWIW: the keyword `query` was from a habit formed from using `react-query` library before 😬